buildOsx.py

```diff
@@ -33,7 +33,7 @@ def build_lang(native_name, cs_name, additional_path, *files):
         "gcc",
         "-fPIC",
         "-shared",
-        "-o", f"{system_lib_dir}/libtree-sitter-{native_name}.{DLYB}",
+        "-o", f"{system_lib_dir}/{get_library_name(native_name, additional_path)}.{DLYB}",
```

Comment:
instead of creating that method, I think we can use os.path and construct the path with each element. By default, if 1 element is null or empty, os.path will not include it when forming the path.